test/functional/cmdline_options_tester/src/TestConfigParser.java

```diff
@@ -191,6 +191,24 @@ public void xmlStartElement(String elementName, Hashtable attributes) {
 					_currentOutput = null;
 				}
 
+			} else if (elementName.equalsIgnoreCase("saveoutput")) {
+				if (hasAllowedPlatform((String)attributes.get("platforms"))) {
+					String regex = (String)attributes.get("regex");
+					String javaUtilPattern = "no";
+					String showRegexMatch = "no";
+					if (null != attributes.get("showMatch")) {
+						showRegexMatch = (String)attributes.get("showMatch");
+					}
+					String caseSensitive = (String)attributes.get("caseSensitive");
+					String saveName = (String)attributes.get("saveName");
+					int splitIndex = Integer.parseInt((String) attributes.get("splitIndex"));
```

Comment:
`splitIndex` may not contain a valid integer value resulting in `NumberFormatException `

We should handle this case and output an error message describing  which element is malformed

Reply:
do you want me to put this exception in a `try catch` ?

Reply:
yes